### DISCUSSION TITLE
[WARNING  ] q-forms of reference and mask are substantially different

### DISCUSSION THREAD

**Discussion by LancyCao0223** [HALFpipe/HALFpipe#671] [Q&A] [answered]
Hello, 

I am seeing these warnings, what does it mean and should I be worry about it? 
[WARNING  ] q-forms of reference and mask are substantially different
[WARNING  ] no contour levels were found within the data range

**Comment by jb21469** [comment]
Hello,

I am also receiving the same warning:
"q-forms of reference and mask are substantially different"

Could anyone provide more information?

Thanks,
Jenna

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @LancyCao0223 @jb21469,

I have also seen these warnings when I use HALFpipe, but they do not seem to correspond to any quality issues, so I think they can be safely ignored.
